src/constants/payment.ts

```diff
@@ -0,0 +1,5 @@
+export const PAYPAL_CLIENT_ID = 'Aa7kWhlwBYR9iBqqv7dBvyudFgkMCQA31wTwpwGjDf-U30efKwIzTHThRwK1hn3iZMHbScoprOIj76SJ'
```

Comment:
Best practice: "Don't push credentials or secret key to repo", instead at least put it to the environment variable 

ex: file .env 

because it can be config in build time by CI and managed and controlled by DevOps team